mock/mock_engine.go

```diff
@@ -381,14 +386,15 @@ func (rme *ResponseMockEngine) findLowestSuccessCode(operation *v3.Operation) st
     return fmt.Sprintf("%d", lowestCode)
 }
 
-func (rme *ResponseMockEngine) lookForResponseCodes(
+func (rme *ResponseMockEngine) findBestMediaTypeMatch(
```

Comment:
nit: was "looking for" media type objects, and the other functions start with "find" so tweaked the name